src/main/java/core/basesyntax/service/impl/FruitReportImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.FruitReport;
+import core.basesyntax.service.FruitService;
+import java.util.List;
+
+public class FruitReportImpl implements FruitReport {
+    private final FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
you don't need this filed here, you should pass ready FruitService with FruitDao inside of it